.travis.yml

```diff
@@ -4,6 +4,10 @@ jdk:
   - oraclejdk7
   - oraclejdk8
 
+script:
+  - mvn clean -Pstrict compile test-compile -B
```

Comment:
install should only clean the compiled class and jar stuff, not the installed stuff